blueprints/banking_blueprint/custom_features.py

```diff
@@ -0,0 +1,41 @@
+from mindmeld.models.helpers import register_query_feature, register_entity_feature
+
+
+@register_query_feature(feature_name="average-token-length")
+def extract_average_token_length(**args):
+    """
+    Example query feature that gets the average length of normalized tokens in the query
+
+    Returns:
+        (function) A feature extraction function that takes a query and
+            returns the average normalized token length
+    """
```

Comment:
Are these feature extractors just examples that aren't used in the app itself? If so, we should just remove them.